src/sql/Expr.cpp

```diff
@@ -1,6 +1,7 @@
 #include "Expr.h"
 #include <stdio.h>
 #include <string.h>
+#include <stdint.h>
```

Comment:
Pick here: Please have a look at the include order. Running the `format.sh`script should already fix this.

I also started a workflow run. Not sure what happened to the gcc-6 stage though. However, we already have a gcc-13 runner there since #242.

Reply:
We might have a problem with Ubuntu 18. But where does node come from? It's from the GitHub action checkout? Curious how that can fail.

https://stackoverflow.com/a/74912791

Reply:
Fixed with #246 
@toge to pass the CI, you need to merge the current master.